### PR TITLE
Error Box Div 

### PR DESCRIPTION
**Pull request by AbnerSD**
Changes can be seen clearly now.

### CODE REVIEW

File: ui/index.html [hunk 1]
```diff
@@ -28,12 +29,24 @@
...
       <div id="ballotEntriesContainer" class="outerContainer"></div>
+      <!-- Error box div to display reportError function when an error occurs -->
+      <div id="errorBox" class="modal" data-backdrop="static" data-keyboard="false">
+         <div class="modal-content">
+            <div class="modal-header">
+               <span class="close" id="modalclose">x</span>
+               <h4> Error Message</h4>
+            </div>
+            <div class="modal-body">
+               <h5 id="Message"></h5>
```
Comment (tom-votes) [inline ui/index.html]: Could use a more-specific id, like maybe `"errorMessage"`

File: ui/index.html [hunk 2]
```diff
@@ -11,7 +12,7 @@
       </div>
       <div id="auditNameContainer" class="container" style="display:none">
          Enter a name for this audit:<br />
-         <input type="text" id="auditNameBox" />
+         <input type="text" id="auditNameBox" maxLength="15" onkeypress="limitKeypress(event,this.value,maxLength)" />
```
Comment (tom-votes) [inline ui/index.html]: I think the `maxLengths` should be longer here to accommodate names like `"bristol_audit_1_district_3_test"` or similar

File: ui/index.html [hunk 3]
```diff
@@ -28,12 +29,24 @@
       </div>
       <div id="seedContainer" class="container" style="display:none">
          Enter random seed:<br />
-         <input type="text" id="seedTextBox" />
+         <input type="text" id="seedTextBox" maxLength="10" onkeypress="limitKeypress(event,this.value,maxLength)" />
```
Comment (tom-votes) [inline ui/index.html]: Some audits use different seeds so maybe we shouldn't limit the `maxLength` here

**Comment by tom-votes** [comment]
Added a few small comments but otherwise looks really good!

File: ui/index.html (hunk 1 above)
Comment (AbnerSD) [reply ui/index.html]: Updated it to errorMessage

File: ui/index.html (hunk 2 above)
Comment (AbnerSD) [reply ui/index.html]: Switched the maxLength to 35

File: ui/index.html (hunk 3 above)
Comment (AbnerSD) [reply ui/index.html]: Removed the 'limitKeypress' function to not restrict input length.

**Comment by tom-votes** [comment]
Looks good, thank you!

**Pull request merged**